source/class/qx/tool/compiler/Analyser.js

```diff
@@ -1415,6 +1415,16 @@ qx.Class.define("qx.tool.compiler.Analyser", {
      * @param library
      */
     addLibrary(library) {
+      if (this.__librariesByNamespace[library.getNamespace()]) {
```

Comment:
Multiple calling of method `getNamespace`. Maybe add some local variable for it?